xtask/Cargo.toml

```
@@ -10,5 +10,5 @@ publish = false
 tonic-build = { version = "0.10", default-features = false, features = [
     "prost", "transport"
 ] }
-clap = { version = "3", features = ["derive"] }
+clap = { version = "4", features = ["derive"] }
```

Since this crate also declares MSRV 1.64, let's use the same version specifier as for `tokio-console`.

```suggestion
clap = { version = "~4.3", features = ["derive"] }
```

---

Fixed.